forest/src/logger/mod.rs

```diff
@@ -0,0 +1,11 @@
+// Copyright 2020 ChainSafe Systems
+// SPDX-License-Identifier: Apache-2.0, MIT
+
+pub(crate) fn setup_logger() {
+    let logger = pretty_env_logger::formatted_timed_builder()
```

Comment:
Also, thoughts on changing the timestamp format
```
2020-02-18T19:35:07.045
```

to something like
```
2020-02-18 19:35:07
```
Nit, but the timestamp is a lot to look at right now lol. Can change in another PR or something, this may just be me